PresentScreenings/Info.plist

```diff
@@ -28,5 +28,9 @@
 	<string>Main</string>
 	<key>XSAppIconAssets</key>
 	<string>Assets.xcassets/AppIcon.appiconset</string>
+	<key>CFBundleDisplayName</key>
+	<string>PlanScreenings</string>
+	<key>NSUserNotificationAlertStyle</key>
+	<string>alert</string>
```

Comment:
Please consider to indicate to set the notification style in System Preferences.
Maybe in a comment in the relevant method in MainClass.

Reply:
Done.